src/main/java/matteroverdrive/client/render/entity/EntityRendererMutantScientist.java

```diff
@@ -39,7 +39,6 @@ public EntityRendererMutantScientist(RenderManager renderManager)
 		super(renderManager, new ModelHulkingScientist(), 0, 1);
 	}
 
-	@Override
```

Comment:
eclipse said that because the code wasn't overriding a class that it should be taken out.
after i did that the client mod ran flawlessly with everything functional that i tested
the server sadly is still trying to run the localization class and my last attempt to fix it broke the client under a nulll pointer when i moved all the code referencing the client class from the main class to the client proxy in an attempt to get the server to run and it crashed both sides. so I undid that change and sent the first working fix to you guys hoping you guys might be able to tell me how to get the server not to crash after getting my small changes to the code.